tools/.cache/token-usage.json

```diff
@@ -0,0 +1,1206 @@
+{
```

Comment:
Is there a possibility we can get away with not committing the cache?

Reply:
Hmm... I'll remove the cache for now and we can build it when we need it in the future.

Reply:
It can be generated on the fly when ever this runs, I just think we can gitignore the cache. There's no need to commit the output either really